iphone/Maps/UI/MailComposer/MailComposer.swift

```diff
@@ -0,0 +1,115 @@
+final class MailComposer: NSObject {
```

Comment:
Is it a refactoring? Can it be merged separately?

Reply:
Sure! Ill make an additional PR